CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 *Enhancements*
 - Officially support Kubernetes 1.14 [#461](https://github.com/Shopify/kubernetes-deploy/pull/461)
+- Allow customising which custom resources are deployed in the pre-deploy phase.
```

Comment:
Can you add a link to the PR here, please? I often find it helpful during releases when going over things in detail.

Reply:
Done